This section is about handling exceptions in python.